### PR TITLE
Switch parsed stashing to new storage for new parsed module implementation

### PR DESCRIPTION
**Pull request by Pchelolo**
cc @wikimedia/service

### CODE REVIEW

**Comment by d00rman** [comment]
LGTM, but let's not merge it until we have the Cassandra 3 tables in place

**Comment by d00rman** [comment]
Hm, Travis does not concur with my +1, lots of `Error: Header \"%s\" value must not be undefined` errors in there.

File: sys/parsoid.js [hunk 1]
```diff
@@ -227,15 +216,7 @@ class ParsoidService {
...
-        // TODO: We're adding the -ng only to avoid table name clashing in dev cluster. Remove!
-        const path = [rp.domain, 'sys', 'key_rev_value', `parsoid.stash.${format}-ng`, rp.title];
-        if (rp.revision) {
-            path.push(rp.revision);
-            if (tid) {
-                path.push(tid);
-            }
-        }
-        return new URI(path);
+        return this.getStashBucketURI(rp, format, tid);
```
Comment (d00rman) [inline sys/parsoid.js]: Hm, the new tables have been created with the `-ng` suffix, so we need to use it here.

**Comment by Pchelolo** [comment]
Fixed node 8 test failure

**Pull request merged**